SMLHelper/Utility/BasicText.cs

```diff
@@ -0,0 +1,435 @@
+﻿/**
+ * BasicText -- Cattlesquat
+ * 
+ * Places a simple text object on the screen and keeps it there until either hidden (or a designated fade-to-black timer has passed). 
+ * By default uses the same font/size/color as the "Press Any Button To Begin" message at the beginning of the game, and appears 
+ * centered about 1/3 down the screen, but all parameters can be reconfigured.
+ * 
+ * The idea of the defaults is that new modders don't have to bootstrap a bunch of irritating Unity stuff -- don't have to understand
+ * what a "Material" is or how to make one, don't have to know to initialize a font, or even a color. Can just start adding text and
+ * then can always custom & configure on further revision.
+ * 
+ * SIMPLE USAGE EXAMPLE:
+ * BasicText message = new BasicText();
+ * message.ShowMessage("This Message Will Fade In 10 Seconds", 10);
+ * 
+ * COMPLEX USAGE EXAMPLE:
+ * BasicText message = new BasicText(TextAnchor.UpperLeft); // Note many other properties could also be set as constructor parameters
+ * message.setColor(Color.red); // Set Color
+ * message.setSize(20);         // Set Font Size
+ * message.setLoc(200, 400);    // Set x/y position (0,0 is center of screen)
+ * message.setFontStyle(FontStyle.Bold); // Bold 
+ * message.ShowMessage("This message stays on screen until hidden"); // Display message; if fadeout seconds not specified, it just keeps showing
+ * ... // other things happen, time goes by
+ * message.Hide(); // Hides the message
+ */
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace SMLHelper.V2.Utility
+{
+    public class BasicText
```

Comment:
TODO: Add XML documentation for Visual Studio.
Simply start typing three forward slashes `///` above any member definition and Visual Studio should auto-complete all the necessary XML tags.

Reply:
I tried what you say in Visual Studio, but nothing happened. I must not have something set up right, but Visual Studio is of course a maze of options.

Reply:
I forgot that's actually not built-in.
Here's the tool that enables that.
https://submain.com/download/ghostdoc/community/

Reply:
![image](https://user-images.githubusercontent.com/3742246/107055052-dae70900-679e-11eb-945b-bc39ed8cff6b.png)
I have Visual Studio 2019 :O